book/src/kubernetes-changelog.md

```diff
@@ -24,6 +45,13 @@ details on individual features, visit the [Features section](features.md).
 * Beta
     * vSphere CSI Migration (on by default)
     * Portworx CSI Migration (off-by-default)
+    * [CSI node expand secret](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3107-csi-nodeexpandsecret)
+    * [Prevent unauthorized volume mode converstion](https://github.com/kubernetes/enhancements/issues/3141)
```

Comment:
Prevent unauthorized volume mode conversion was added as alpha in 1.24, moved to beta in 1.27.